### PR TITLE
set up storybook

### PR DESCRIPTION
**Pull request by winston0410**

### CODE REVIEW

**Review by vadim-kirpun — CHANGES_REQUESTED** [review]
This PR contains a lot of files not related to our projects
As the first step of Storybook integration let's create a story only for the `GrayBox` component which is already added to this library. Let's keep this story in the component's folder

**Comment by vadim-kirpun** [comment]
image

I see lots of TS errors in `Button.stories.tsx` and other files
Please make sure husky works on precommit. If it misses some errors please adjust eslint configuration so it catches all the errors

**Comment by winston0410** [comment]
> image
> 
> I see lots of TS errors in `Button.stories.tsx` and other files Please make sure husky works on precommit. If it misses some errors please adjust eslint configuration so it catches all the errors

Are you using a global eslint configuration in your VSCode? If thats the case, I will not be able to debug that because I cannot see any error when I open the file.

**Comment by vadim-kirpun** [comment]
All good!
As the last step let's move this story to `GrayBox` folder so we keep everything related to the component in the same place (e.g. styles, story, tests)

File: src/stories/GrayBox.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import GrayBox from '../components/layout/GrayBox/GrayBox';
+
+export default {
+  title: 'Example/GrayBox',
```
Comment (vadim-kirpun) [inline src/stories/GrayBox.stories.tsx:6]: Let's name this section `Layout` instead of `Example`

**Review by vadim-kirpun — CHANGES_REQUESTED** [review]

**Review by vadim-kirpun — APPROVED** [review]

**Pull request merged**